uPortal-webapp/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -148,6 +148,10 @@
 <script id="backgroundScript">
 (function(){
```

Comment:
:notebook: N.B. a common pattern within the portal is to use an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) with `up.jQuery` passed as a param.

E.G.
```js
(function ($) {
   // some business logic here
})(up.jQuery);

Reply:
This is a great pattern.  :+1: